app/routes/http/AuthRoute.ts

```diff
@@ -0,0 +1,7 @@
+import Route from '@ioc:Adonis/Core/Route'
+
+
+Route.post('/register',
+    'AuthController.registerUser').prefix('auth');
+Route.post('/login',
+    'AuthController.loginUser').prefix('auth');
```

Comment:
Group this route and add a prefix to that group so we don't have to add the prefix `auth` in every route.